### PR TITLE
[feature] automate PyPi package publishing with GitHub Actions

### PR DESCRIPTION
**Pull request by omadson**
Hi @zealberth. I create a workflow to automate the pypi package publishing on a github release. Now, every time you create a release on github, the package will be published in pypi with the release tag. Of course, you have to create an acount on pypi. Another thing you have to do is create two secret variables:
 - `PYPI_USERNAME` : with your pypi username as value
 - `PYPI_PASSWORD`: with your pypi password as value

in short, I made these changes:
 - created a workflow
 - modified the pyprojet file

### CODE REVIEW

**Comment by omadson** [comment]
don't merge this pr yet!

I'll put the information about the new installation method in `README.md`.

**Review by zealberth — APPROVED** [review]
Everything is ok!
